### DISCUSSION TITLE
Option to restore just selected JSONs

### DISCUSSION THREAD

**Discussion by ztrhgf** [almenscorner/IntuneCD#145] [Ideas]
It would be nice to have an option to restore just selected JSON backup files. Not a whole backup. If I am not mistaken thats not possible right now?

**Comment by almenscorner** [comment]
If you work with prefixes it is possible to backup and update only configs you want to keep track of, it is also possible to exclude payloads by using `-e ConditionalAccess` for example.

If you mean point out specific files in a full backup, no, that is not possible. On the other hand, if you only care about certain files and know which ones you could just move those to another folder and run the update from there.

**Reply by ztrhgf** [reply]
OK, so just keep the folder structure, with the required JSON(s) and use  `--path` parameter

**Reply by almenscorner** [reply]
Yes! that would then only update/create those configurations
